public/css/app.css

```diff
@@ -53,9 +53,14 @@ body, html {
 }
 
 .footer {
+    position: fixed;
```

Comment:
Why do we have a fixed position here? It leaves very little room for actual content (and hides content in some cases). I could not test the rest of user auth because the registration button is hidden.